js/search/search.js

```diff
@@ -326,9 +326,10 @@ class Search {
      * @param {String} searchQuery
      * @param {Array} senderId
      * @param {Array} threadId
+     * @param {String} fileType
      * @returns {string}
      */
-    static constructQuery(searchQuery, senderId, threadId) {
+    static constructQuery(searchQuery, senderId, threadId, fileType) {
 
         let query = "";
```

Comment:
Can you refactor this to something like "searchText" as it's confusing with all the other names like searchQuery, etc.?